pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -111,8 +111,14 @@ func newWorkflowConfig() *workflowConfig {
 }
 
 func (o *capability) RegisterToWorkflow(ctx context.Context, request capabilities.RegisterToWorkflowRequest) error {
+	err := o.ValidateConfig(request.Config)
```

Comment:
```suggestion
	config, err := o.UnwrapAndValidateConfig(request.Config)
```

Reply:
I suspect that in most cases you'll want to unwrap before validating

Reply:
nice, having unwrap + validate in a single call would be great for DX